js/app.js

```diff
@@ -87,12 +118,80 @@ function calculoDelTotal() {
             let precioEn12 = sumaTotal / 12;
             alert(`El precio total le queda en 12 cuotas de ${precioEn12}`);
             break;
-        case "18":
+            case "18":
             let precioEn18 = sumaTotal / 18;
             alert(`El precio total le queda en 18 cuotas de ${precioEn18}`);
             break;
-        default:
-            alert(`Su monto a pagar es ${sumaTotal} pesos en un solo pago`);
+            default:
+                alert(`Su monto a pagar es ${sumaTotal} pesos en un solo pago`);
             break;
     }
+}
+*/
+
+let stockNeumaticos = [
+    {id: 1,marca: "Bridgestone", modelo:"EP150", medida:"195/55R16", tipo: "auto", descripcion: "", cantidad: 1,stock: 4, precio: 14000,  displonible: true, img: `./assets/imagenVacia.jpg`},
+    {id: 2,marca: "Bridgestone", modelo:"Turanza ER300", medida:"205/55R16", tipo: "auto", descripcion: "", cantidad: 1,stock: 8, precio: 21000, displonible: true, img: `./assets/imagenVacia.jpg`},
+    {id: 3,marca: "Firestone", modelo:"FH900", medida:"195/65R15", tipo: "auto", descripcion: "", cantidad: 1,stock: 2, precio: 21000, displonible: true, img: `./assets/imagenVacia.jpg`},
+    {id: 4,marca: "Bridgestone", modelo:"H/T 684", medida:"265/65R17", tipo: "camioneta", descripcion: "", cantidad: 1,stock: 6, precio: 45000, displonible: true, img: `./assets/imagenVacia.jpg`},
+    {id: 5,marca: "Firestone", modelo:"CV 5000", medida:"195/75R16", tipo: "camioneta", descripcion: "", cantidad: 1,stock: 10, precio: 40000, displonible: true, img: `./assets/imagenVacia.jpg`},
+    {id: 6,marca: "Bridgestone", modelo:"M/T 674", medida:"265/65R17", tipo: "camioneta", descripcion: "", cantidad: 1,stock: 12, precio: 75000, displonible: true, img: `./assets/imagenVacia.jpg`}, 
+]
+
+let carrito =[]
+
+const contenidoCarrito = document.querySelector(`#contenidoCarrito`)
+
+ const listaProductos = document.querySelector(".wrapProducts")
+
+ const contadorCarrito = document.querySelector(`#contadorCarrito`)
```

Comment:
Inconsistencia en las comillas al usar string. En JS para los strings se usan comillas simples.
```suggestion
const contenidoCarrito = document.querySelector('#contenidoCarrito')
const listaProductos = document.querySelector('.wrapProducts')
const contadorCarrito = document.querySelector('#contadorCarrito')
```